roles/servicetelemetry/templates/manifest_snmp_traps.j2

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: snmp-webhook
+spec:
+  selector:
+    matchLabels:
+      app: snmp-webhook
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: snmp-webhook
+    spec:
+      containers:
+        - name: prometheus-webhook-snmp
+          image: 'quay.io/mrunge/prometheus-webhook-snmp:latest'
```

Comment:
This will need to be changed to point at a container image managed in the infrawatch namespace (or other)

Reply:
Yes, but for now, it is not. I'm more than happy to move it somewhere else.

Reply:
The issue is that we also need this to get packaged downstream if we're going to merge it into master, which means we need to start the process with release delivery now so we can have it for STF 1.1